src/commands/exec.yml

```
@@ -24,3 +24,6 @@ steps:
       command: secrethub run << parameters.flags >> -- $SHELL -c '<< parameters.command >>'
       environment:
         SECRETHUB_RUN_NO_PROMPT: true
+        SECRETHUB_APP_INFO_NAME: secrethub-circleci-orb
+        # Version gets updated automatically
```

Slightly more descriptive:
```suggestion
        # Version is automatically bumped on release branches
```